jax/experimental/jax2tf/jax2tf.py

```diff
@@ -56,11 +56,17 @@ def _is_tfval(v: TfVal) -> bool:
   if isinstance(v, (tf.Tensor, tf.Variable)):
     return True
   try:
-    tf.convert_to_tensor(v)
+    safe_convert_to_tensor(v)
```

Comment:
Please add a comment here that this conversion is overkill just for a type check, but this code is only run if core.skip_checks is False. (I hope the latter is true)

Reply:
Added a comment